buf.gen.yaml

```diff
@@ -1,4 +1,4 @@
-version: v1beta1
+version: v1
```

Comment:
Why?

Reply:
this is an update - v1 is a released after the v1beta1.

Reply:
Understood